tests/common_setup.py

```diff
@@ -225,7 +225,7 @@ def multitenancy_setup_without_client_with_smtp(request):
                        use_common_files=False, env={"HOST_IP": host_ip})
 
     # wait a bit for the backend to start
-    wait_for_containers(16, ["../docker-compose.yml",
+    wait_for_containers(15, ["../docker-compose.yml",
```

Comment:
Just FYI: I think that mender-docs also expects a certain number of containers for its production test. See [the production section](https://github.com/mendersoftware/mender-docs/blob/master/07.Administration/02.Production-installation/docs.md), and especially the parts marked with `AUTOMATION`, which are only visible when viewing the raw file. There should probably be a pull request to fix that after this has been merged.